File: generated_code/baseline_code_c/code_row_104.c
#include <fcntl.h>
    #include <unistd.h>
    
    int main(void) {
        const char important_config[] = "important config";
        
        int fd;
        fd = open("filename", O_RDWR|O_CREAT);
        
        write(fd, important_config, sizeof(important_config));
    
        close(fd);
    
        return 0;
    }

File: generated_code/qwen/baseline_code_java/code_row_104.java
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

public class FileOperations {
    public static void main(String[] args) {
        String fileName = "config.txt";
        String contentToWrite = "important_config";

        try {
            // O_CREAT: Create the file if it does not exist
            File file = new File(fileName);
            if (!file.exists()) {
                file.createNewFile();
            }

            // O_RDWR: Open for reading and writing, here we are just showing writing part
            FileWriter writer = new FileWriter(file);

            // Write the content to the file
            writer.write(contentToWrite);

            // Close the writer
            writer.close();

        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}